src/control/server/ctl_storage_rpc_test.go

```diff
@@ -517,6 +541,26 @@ func TestServer_CtlSvc_StorageScan_PostEngineStart(t *testing.T) {
 				Scm: &ctlpb.ScanScmResp{State: new(ctlpb.ResponseState)},
 			},
 		},
+		"engine up; scan bdev basic; vmd enabled": {
```

Comment:
Hm, do we need this additional case? Is the PCI addr of a VMD device the only thing different?

Not that I'm opposed to additional coverage, but if the logic isn't different for VMD, it may be duplicating the test case above.

Reply:
yes agree it's not necessary, I added when I thought there was going to be slightly different logic

Reply:
done